CodeSignal/dockerfiles
# Install `split` for Haskell.

Install `split` for Haskell to make `Data.List.Split` available.

I already deployed a coderunner `104.248.224.89` on production rundash, if you ssh into it, you can see that its haskell docker image is changed to `install-split`. 

## To test:
Go to the main app, and sign in as admin, and go to http://localhost:3000/admin/configs. Add `"targetHost": "104.248.224.89"` in `rundashApi`.
Now go to http://localhost:3000/arcade/intro/level-1/SJnHWoCmcLneruiug, try to run this code:
```hs
import Data.List.Split
add param1 param2 = unsafePerformIO $ do
    print $ (show param1) ++ (show param2)
    return $ param1 + param2
```
You should see no errors and pass all tests! 🎉
